definitions/artifacts/elastic-authentication.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "https://api.massdriver.cloud/artifact-definitions/massdriver/elastic-authentication.json",
+  "$md": {
+    "access": "public"
+  },
+  "type": "object",
+  "title": "Elasticsearch Cluster",
+  "description": "Elasticsearch cluster authentication and cloud-specific configuration.",
+  "additionalProperties": false,
+  "required": [
+    "data"
+  ],
+  "properties": {
+    "data": {
+      "type": "object",
+      "required": [
+        "infrastructure",
+        "authentication"
+      ],
+      "properties": {
+        "infrastructure": {
+          "$ref": "../types/k8s-application-infrastructure.json"
+        },
+        "authentication": {
+          "$ref": "../types/elastic-authentication.json"
+        }
+      }
+    }
+  }
```

Comment:
We'll want a `specs` property as well (and we can add it to required). I know I've echo'd "we don't really use specs, they might go away" but after the chat in `bundles-discuss` it looks like we'll be using them more going forward. Additionally, they're required by the backend, even if they're empty!

[Lore chat](https://massdriverworkspace.slack.com/archives/C03HMFZU4UF/p1655404115374629)


Here's the [rdbms](https://github.com/massdriver-cloud/artifact-definitions/blob/main/definitions/specs/rdbms.json) spec file as an example.

Reply:
I just don't know which spec to use. Make a `version` spec for elastic maybe?